huntsman/utils/pyro/camera_server.py

```diff
@@ -9,7 +9,10 @@
 from huntsman.camera.pyro import CameraServer
 
 
-def run_camera_server(ignore_local=False, unmount_sshfs=True, logger=None,
+def run_camera_server(ignore_local=False,
+                      key=None,
```

Comment:
I assumed this was an ssh key but it looks like it is a config key item?  Maybe be more explicit and add to docstring.

Reply:
Or maybe just pass it through via `kwargs` if `run_camera_server` itself doesn't need it. Although that's a bit hiding it.

Reply:
Yes, config key. This parameter got added along with the config server stuff, but when I tried to actually use it to run a local camera server for testing purposes I discovered it didn't actually work. My changes in this file were the result of my attempt to fix it so that I could use a local camera server to work out some tricky bugs in the Pyro camera stuff. After a fair bit of wasted time poking at this stuff I discovered that it wasn't going to work anyway because there is no way to pass the `key` parameter to the `CameraServer` instance, and that will always try to use a config for the host's IP address regardless. That's why the config server test fixture in `conftest` inserts a device config with the host's IP address on the fly, and why I ended up added the `localhost` fallback in `huntsman.utils.config.load_device_config`. I should just remove all traces of the `key` parameter since it's not used and can't work.